buildrpm/ocne.spec

```diff
@@ -6,7 +6,7 @@
 
 Name: ocne
```

Comment:
Should this file not be changed on this PR?

Reply:
I think changelog should be updated here